src/events/mqtt.js

```diff
@@ -0,0 +1,109 @@
+const shimmer = require('shimmer');
+const tracer = require('../tracer.js');
+const eventInterface = require('../event.js');
+const moduleUtils = require('./module_utils.js');
+
+
+const getPublishParams = (topic, message, options, callback) => {
+    const topic_internal = topic;
+    const message_internal = message;
+    const options_internal = options;
+    const callback_internal = callback;
+
+    return {
+        topic_internal,
+        message_internal,
+        options_internal,
+        callback_internal
+    };
+};
```

Comment:
no need for this if there is no messing with the parameters :)
we are doing this in other libraries since sometimes the callback is passed at a different argument (cause options is skipped for example) and we need to locate it. The logic here seems simple (unless options can be skipped, then this is an issue we should address)